src/clj_kondo/impl/analyzer.clj

```diff
@@ -1604,6 +1604,17 @@
                              (node->line filename f
                                          :invalid-arity
                                          (linters/arity-error nil fn-name arg-count fixed-arities varargs-min-arity))))))))))
+    (when (and (not (utils/linter-disabled? ctx :two-argument-reduce))
+               (= 'reduce resolved-as-name)
```

Comment:
This should probably be resolved-name

Reply:
I mean, the originally resolved-name from analyze-call, not the one that is in scope here

Reply:
afaict it's the same `resolved-as-name` that is being used to determine transducer eligibility, see:  https://github.com/clj-kondo/clj-kondo/blob/master/src/clj_kondo/impl/analyzer.clj#L1556
it seems to be i'm using it in a consistent manner (which may still be wrong of course)

Reply:
fair enough. it's just that when people lint something as `reduce`, they will get this warning for this linter.

Reply:
that's a good point, this linter is just super specific...